contents/blog/five-essential-posthog-apps.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2022-09-10
+title: "Getting started: Your five essential PostHog apps"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["joe-martin"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+categories: ["Guides"]
+---
+
+PostHog apps are a powerful, but ill-defined part of the platform. They’re powerful because they can do almost anything — and they’re ill-defined because they do almost anything.
```

Comment:
```suggestion
PostHog apps are a powerful, but hard to explain part of the platform. They’re powerful because they can do almost anything — and they’re hard to explain because they do almost anything.
```